docs/getting_started/getting_started_chat.md

```diff
@@ -0,0 +1,217 @@
+# Chat Models
+
+This guide walks you through how the core LangChain modules work with chat models. We recommend first reading through the [Quickstart Guide](./getting_started.md), which introduces the core concepts in the context of LLMs.\
+\
+Chat models are a variation on language models. While chat models use language models under the hood, the interface they expose is a bit different: rather than expose a "text in, text out" API, they expose an interface where "chat messages" are the inputs and outputs.\
+\
```

Comment:
Are the backslashes needed if we do double newlines?

Reply:
yea sphynx (or our theme or whatever, i don't really understand these things) doesn't give a damn how many new lines i add unfortunately